src/main/java/com/teamc/fms/service/impl/FacilityServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.teamc.fms.service.impl;
+
+import com.teamc.fms.entity.Facility;
+import com.teamc.fms.repo.FacilityRepository;
+import com.teamc.fms.service.FacilityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class FacilityServiceImpl implements FacilityService {
+
+    @Autowired
+    private FacilityRepository facilityRepository;
+
```

Comment:
Please use constructor injection and Lombok to avoid biolerplate code, like the attached image